cache/v0.1/cache-v0.1.md

```diff
@@ -0,0 +1,22 @@
+# cache v0.1
+
+<h2>Directives related to caching</h2>
+
+```raw html
+<table class=spec-data>
+  <tr><td>Status</td><td>Release</td>
+  <tr><td>Version</td><td>0.1</td>
+</table>
+<link rel=stylesheet href=https://specs.apollo.dev/apollo-light.css>
+<script type=module async defer src=https://specs.apollo.dev/inject-logo.js></script>
+```
+
+This specification provides a list of directives related to caching. 
```

Comment:
Can we include a note here that this is taken from https://www.apollographql.com/docs/apollo-server/performance/caching/?